webvtt.go

```diff
@@ -24,6 +25,7 @@ const (
 // Vars
 var (
 	bytesWebVTTTimeBoundariesSeparator = []byte(webvttTimeBoundariesSeparator)
+	voiceTagRegexp                     = regexp.MustCompile(`(<v([\.\w]*)([\s\w]+)+>)*(.[^<]*)(</v.*>)*`)
```

Comment:
Could you rename this variable to `webVTTRegexpVoiceName` ?

Reply:
yes